doc/classes/Camera3D.xml

```diff
@@ -218,10 +218,10 @@
 			Disables [url=https://en.wikipedia.org/wiki/Doppler_effect]Doppler effect[/url] simulation (default).
 		</constant>
 		<constant name="DOPPLER_TRACKING_IDLE_STEP" value="1" enum="DopplerTracking">
-			Simulate [url=https://en.wikipedia.org/wiki/Doppler_effect]Doppler effect[/url] by tracking positions of objects that are changed in [code]_process[/code]. Changes in the relative velocity of this camera compared to those objects affect how Audio is perceived (changing the Audio's [code]pitch shift[/code]).
+			Simulate [url=https://en.wikipedia.org/wiki/Doppler_effect]Doppler effect[/url] by tracking positions of objects that are changed in [code]_process[/code]. Changes in the relative velocity of this camera compared to those objects affect how Audio is perceived (changing the Audio's [member AudioStreamPlayer3D.pitch_scale]).
```

Comment:
For the record, I double checked and this is indeed what seems to be done in `AudioStreamPlayer3D::_update_panning()` :+1: 

I'd change the uppercased "Audio" to "audio" though (here and below, might have other similar occurrences).

Reply:
I should have been more explicit, there's another two occurrences of "Audio" in those two descriptions.

Reply:
My bad. I noticed those two "Audio"s, but thought they were referring to something else :joy: